relay-dynamic-config/src/defaults.rs

```diff
@@ -136,16 +138,17 @@ fn span_metrics(transaction_extraction_enabled: bool) -> impl IntoIterator<Item
         & RuleCondition::glob("span.op", "app.start.*")
         & RuleCondition::eq("span.description", APP_START_ROOT_SPAN_DESCRIPTIONS);
 
-    // `exclusive_time_light` is the metric with the most lenient condition.
+    // `exclusive_time_light` excludes transaction tags (and some others) to reduce cardinality.
     let exclusive_time_light_condition = (is_db.clone()
+        | is_ai.clone()
```

Comment:
This will start recording `exclusive_time` and `duration` metrics, is that what you want as well? Do we need both the light and normal `exclusive_time` ones? This light one was added for starfish related work so unless you plan to query on both, you might not need that.